meson.build

```diff
@@ -0,0 +1,53 @@
+project(
+  'hidapi',
+  'c',
+  version : '0.12.0',
+  default_options : [ 'warning_level=1', ],
+  meson_version : '>=0.60',
+)
+
+iconv_dep = dependency('iconv', required: get_option('libusb'))
+libusb_dep = dependency('libusb-1.0', required: get_option('libusb'))
+udev_dep = dependency('libudev', required: get_option('hidraw'))
+
+incdirs = include_directories('hidapi')
+
+install_headers('hidapi/hidapi.h', subdir: 'hidapi')
+
+pconf = import('pkgconfig')
+
+if libusb_dep.found() and iconv_dep.found()
+  libhidapi = library('hidapi-libusb',
+    'libusb/hid.c',
+    dependencies: [ iconv_dep, libusb_dep ],
```

Comment:
This should depend on threads, I think.

Also, it is only called "-libusb" on linux.

And on Windows or macOS it uses the sources from windows/ or mac/ instead. See e.g. https://github.com/libusb/hidapi/blob/59e84ca74fae41a43f8ff3ea96b4f956c79c69b4/Makefile.am#L18-L40

Reply:
threads probably implicitly gets included with libusb.